### PR TITLE
Add group directly in au

### PR DESCRIPTION
**Pull request by SwissGipfel**

### CODE REVIEW

File: msgraph/administrative_units_test.go [hunk 1]
```diff
@@ -177,6 +236,17 @@ func testAdministrativeUnitsClient_RemoveMembers(t *testing.T, c *test.Test, adm
 	}
 }
 
+func testAdministrativeUnitsClient_CreateGroup(t *testing.T, c *test.Test, administrativeUnitId string, g *msgraph.Group) (group *msgraph.Group) {
+	group, status, err := c.AdministrativeUnitsClient.CreateGroup(c.Context, administrativeUnitId, g)
+	if err != nil {
+		t.Fatalf("AdministrativeUnitsClient.CreateGroup(): %v", err)
+	}
+	if status < 200 || status >= 300 {
```
Comment (ccadruvi) [inline msgraph/administrative_units_test.go:244]: We expect status code 201, so anything else should be an error, right?
```suggestion
	if status < 201 || status > 201 {
```

File: msgraph/administrative_units_test.go [hunk 2]
```diff
@@ -11,6 +11,53 @@ import (
 	"github.com/manicminer/hamilton/odata"
 )
 
+//func TestCreateGroup(t *testing.T) {
```
Comment (ccadruvi) [inline msgraph/administrative_units_test.go:14]: This test can be removed before merging.

Comment (SwissGipfel) [reply msgraph/administrative_units_test.go]: removed

File: msgraph/administrative_units_test.go (hunk 1 above)
Comment (SwissGipfel) [reply msgraph/administrative_units_test.go]: rewritten:

```go
if status != http.StatusCreated {
```

**Review by ccadruvi — APPROVED** [review]

**Pull request merged**